Makefile

```diff
@@ -393,6 +393,15 @@ test-client:
 	rm -rf src/client/vendor
 	git checkout src/server/vendor/github.com/pachyderm
 
+test-vault:
+	rm -rf src/server/vendor/github.com/pachyderm/src/client
```

Comment:
why delete this reference? Does it break start-vault?

Reply:
Hrmm, at some point I was getting a funny import error. But removing it and it doesn't happen. So I've removed this line (and the corresponding cleanup line).